pyproject.toml

```diff
@@ -26,6 +26,7 @@ cryptography = ">=41.0.6,<42.1.0"
 nats-py = { version = "2.6.0", optional = true }
 boto3 =  { version = ">=1.28.16,<1.29.0", optional = true }
 kafka-python = { version = ">=2.0.0,<2.1.0", optional = true }
+amqpstorm = {version = "^2.10.7", optional = true, extras = ["amqpstorm"]}
```

Comment:
Just curious what was the decision process to pick this?
As in why not the more popular pika library? or celery's pyamqp?